docs/user-guide/address-network-requirements.md

```diff
@@ -0,0 +1,18 @@
+# Address network requirements
+
+When installing Zowe server-side components, review the following table to determine which ports are required. 
+These are default values. You can change the values by updating variable values in the `zowe.yaml` file. 
```

Comment:
Link to the top of https://docs.zowe.org/stable/appendix/zowe-yaml-configuration to explain what those "variable names" mean.
by the way, the zowe-yaml-configuration page itself should be redesigned now, to be more useful for this purpose. people still dont understand yaml well, and we're jumping right into telling people to edit it

Reply:
link added